### DISCUSSION TITLE
Which communicator should I use as parameter for `PETSc.garbage_cleanup`?  `mesh.comm`  or `mesh._comm`?

### DISCUSSION THREAD

**Discussion by lrtfm** [firedrakeproject/firedrake#2858] [Firedrake support]
1. What's the difference between `mesh.comm` and `mesh._comm`?
2. Is calling `PETSc.garbage_cleanup` with `mesh.comm` equivalent to calling it with `mesh._comm`?

**Comment by connorjward** [comment]
@JDBetteridge?
